apps/mark/frontend/test/helpers/mock_api_client.tsx

```diff
@@ -73,6 +80,10 @@ export function createApiMock() {
     );
   }
 
+  const electionStateRef: { current: ElectionState } = {
+    current: initialElectionState,
+  };
```

Comment:
Any reason to keep this state as a variable? It looks like we only modify this state right before setting the resolved value in `expectGetElectionState`, which already has access to the desired resolved value as an argument. Would it be cleaner to just pass the desired resolved value as an argument like `setUsbDriveStatus`?

Reply:
It maintains a history of "last election state" which allowed me, when writing tests, to only pass in the "diff" of the election state. If we just passed the desired resolved value in a test where the polls status were changing, you'd have pass the `precinctSelection` in every time as well. I think there is some clarity to that but it's also an annoyance and a footgun of a different kind.

Reply:
Ah your comment below makes sense now, thanks